lib/survey_execution/initializer.rb

```diff
@@ -0,0 +1,12 @@
+module SurveyExecution
+  class Initializer
+    def initialize(survey, respondents)
+      @client = Sms::Client.new
```

Comment:
I was considering making it a singleton previously, but I though I read somewhere that singletons in rails is an anti-pattern. But researching again I think it's okay to use a singleton for this purpose.